symengine/polys/upolybase.h

```diff
@@ -271,6 +271,17 @@ RCP<const Poly> mul_upoly(const Poly &a, const Poly &b)
     dict *= b.get_poly();
     return Poly::from_dict(a.get_var(), std::move(dict));
 }
+
+template <typename Poly>
+RCP<const Poly> quo_upoly(const Poly &a, const Poly &b)
```

Comment:
@isuruf We need a `div_upoly` function also, which can return both quotient and remainder.